.env.example

```
@@ -1,8 +1,18 @@
-# .env.example
+# ── OpenDota API ────────────────────────────────────────────────
 OPENDOTA_BASE_URL=https://api.opendota.com/api
-OPENDOTA_RATE_LIMIT=60
-OPENDOTA_TIMEOUT=30
-OPENDOTA_RETRIES=3
+OPENDOTA_RATE_LIMIT=60    # requests per minute (free tier = 60)
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)